### PR TITLE
add "limit" option to search

### PR DESCRIPTION
**Pull request by garak**
See #65

### CODE REVIEW

**Review by robbieaverill — COMMENTED** [review]
I wonder whether passing `$limit` as an integer might be better, so you could specify how many results you want?

**Comment by garak** [comment]
> I wonder whether passing `$limit` as an integer might be better, so you could specify how many results you want?

It makes sense, but it would pick a number of *pages*, not a number of results

**Comment by robbieaverill** [comment]
Sounds good 👍

**Comment by garak** [comment]
Changed (and squashed)

File: src/Packagist/Api/Client.php [hunk 1]
```diff
@@ -68,10 +68,11 @@ public function __construct(
      *
      * @param string $query   Name of package
      * @param array  $filters An array of filters
+     * @param int    $limit   Pages tolimit results (0 = all pages)
```
Comment (JellyBellyDev) [inline src/Packagist/Api/Client.php:71]: `tolimit` => `to limit `?

**Review by JellyBellyDev — APPROVED** [review]

**Review by robbieaverill — APPROVED** [review]

**Comment by robbieaverill** [comment]
It's likely that I'll bundle this together with #64, to be honest. Does that sound okay to you?

**Pull request merged**